packages/2018-disaster-resilience/src/components/EarthquakeDamageEstimatesForBuildingsInTillamookCounty/index.js

```diff
@@ -12,13 +12,10 @@ const EarthquakeDamageEstimatesForBuildingsInTillamookCounty = ({
   data,
   Layout
 }) => {
-  useEffect(
-    () => {
-      init();
-      // eslint-disable-next-line react-hooks/exhaustive-deps
-    },
-    [init]
-  );
+  useEffect(() => {
+    init();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [init]);
```

Comment:
Remove init, make the eslint disable the same line. Might also increase our speed